ch03.md

```diff
@@ -217,7 +217,8 @@ In fact, we find the functional community pioneering new test tools that can bla
 
 Many believe the biggest win when working with pure functions is *referential transparency*. A spot of code is referentially transparent when it can be substituted for its evaluated value without changing the behavior of the program.
 
-Since pure functions always return the same output given the same input, we can rely on them to always return the same results and thus preserve referential transparency. Let's see an example.
+Since pure functions don't have side effects, they can only influence the bevhavior of a program through their output values. Furthermore, since their output values can reliably be calculated using only their input values, pure functions will always preserve referential transparency. Let's see an example.
```

Comment:
I like it :)! 
There's a minor typo however: be**v**havior --> behavior

Reply:
boom! Great catch! Fixed :D